web/src/app/escalation-policies/PolicyStepForm.js

```diff
@@ -25,11 +25,12 @@ import { SlackBW as SlackIcon } from '../icons/components/Icons'
 import { Config } from '../util/RequireConfig'
 import NumberField from '../util/NumberField'
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   badge: {
     top: -1,
     right: -1,
-    backgroundColor: '#cd1831',
+    // TODO if practical, use themed color for light mode
+    backgroundColor: theme.palette.mode === 'dark' ? 'secondary' : '#cd1831',
```

Comment:
we should hold off on this and use the theme color, as this will just have to change again soon if we merge

Reply:
It's just double work 🤷